draco/core/containers.py

```diff
@@ -3340,3 +3340,227 @@ def copy_datasets_filter(
+            "dtype": np.complex128,
+            "initialise": True,
+            "distributed": False,
+        },
+    }
+
+    @property
+    def comp_bandpass(self):
+        """Get the comp_bandpass dataset."""
+        return self.datasets["comp_bandpass"]
+
+    @property
+    def sval(self):
+        """Get the sval dataset."""
+        return self.datasets["sval"]
+
+    
+class VisBandpassWindowBaselineRA(FreqContainer):
+    """Container for bandpass gains and their window estimated by running bandpass HyFoReS on hybrid beam-formed visibilities"""
+
+    _axes = ("pol", "ew", "ra", "freq") #Q8: do I need to declare _axes? What is the point of this? might have to add pol                                                                                                                                                                                                                                                               
+
+    _dataset_spec: ClassVar = {
+        "bandpass": {
+            "axes": ["pol", "ew", "ra", "freq"],
+            "dtype": np.complex128,
+            "initialise": True,
+            "distributed": True,
+            "distributed_axis": "ra",
+            "chunks": (1, 4, 2048, 32),
+            "compression": COMPRESSION,
+            "compression_opts": COMPRESSION_OPTS,
+        },
+        "window": {
+            "axes": ["pol", "ew", "ra", "freq", "freq"],
+            "dtype": np.complex128,
+            "initialise": True,
+            "distributed": True,
+            "distributed_axis": "ra",
+            "chunks": (1, 4, 2048, 32, 32),
+            "compression": COMPRESSION,
+            "compression_opts": COMPRESSION_OPTS,
+        },
+    }
+
+    #_data_dset_name = "stack"                                                                                                                                                                                                                                                                                                                                              
+    #_weight_dset_name = "weight"                                                                                                                                                                                                                                                                                                                                           
+
+    @property
+    def bandpass(self):
+        """Get the bandpass dataset."""
+        return self.datasets["bandpass"]
+
+    @property
+    def window(self):
+        """Get the window dataset."""
+        return self.datasets["window"]
+
+class VisBandpassCompensateBaselineRA(FreqContainer):
```

Comment:
Same comment as above with inheriting from `VisBandpassCompensateBaseline` and `SiderealContainer`